netbox/templates/500.html

```diff
@@ -14,9 +14,9 @@
         <div class="row">
             <div class="col col-md-6 offset-md-3">
                 <div class="card border-danger mt-5">
-                    <h5 class="card-header">
+                    <h2 class="card-header h5">
```

Comment:
You're correct in your assumption that the heading elements are used in cases such as this one solely for styling. If there's a need to enforce sequential ordering of headings (and I'm not sure there is?), it would make more sense IMO to change this to a `<div>`. Keeping these as `<h2>` tags with the `h5` class is confusing.

More to the point, for card headers specifically, we should just tweak the `card-header` class as needed to obviate the need for the extra tag.

Reply:
In the same way that sighted users can glance at a page and get a sense of its contents, users of screen readers can do the same by navigating through headings. Clear and sequential headings save those who use screen readers time and frustration. [Source](https://dequeuniversity.com/rules/axe/4.9/heading-order) provided by google lighthouse.

I do agree having a `<h2>` and `.h5` together is odd, it was the best way to ensure the cards I was targeting will keep the correct style where as altering `.card-header` may be used in other areas where we do not want to make styling changes such as the dashboard cards.